tendrl/ceph_integration/objects/rbd/atoms/check_rbd_available/__init__.py

```diff
@@ -0,0 +1,46 @@
+import etcd
+import gevent
+
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons import objects
+
+
+class CheckRbdAvailable(objects.BaseAtom):
+    def __init__(self, *args, **kwargs):
+        super(CheckRbdAvailable, self).__init__(*args, **kwargs)
+
+    def run(self):
+        retry_count = 0
+        while True:
+            try:
+                rbd = NS._int.client.read(
+                    "clusters/%s/Pools/%s/Rbds/%s" %
```

Comment:
use object.load() , we need to ensure all attributes inside the object are also present as provided by user